app/controllers/katello/api/v2/errata_controller.rb

```diff
@@ -20,8 +20,12 @@ def available_for_content_view_filter(filter, collection)
       collection = filter_by_content_view(filter, collection)
       ids = Katello::ContentViewErratumFilterRule.where(:content_view_filter_id => filter.id).pluck("errata_id")
       collection = collection.where("errata_id not in (?)", ids) unless ids.empty?
-      collection = collection.where('issued  >= ?', params[:start_date]) if params[:start_date]
-      collection = collection.where('issued  <= ?', params[:end_date]) if params[:end_date]
+      date_type = "issued"
+      if params[:use_issued_date].present? && params[:use_issued_date] == false
```

Comment:
Looks like that was by accident:

https://github.com/parthaa/katello/commit/9242ac769ebd01f5a3991a25fa78dcec3c5823c9#diff-234495ca5be1bd137b51fa1e28aa2cdc